Misc/NEWS.d/next/Library/2022-10-02-12-38-22.gh-issue-82836.OvYLmC.rst

```diff
@@ -0,0 +1 @@
+Fix implemented for Lib/ipaddress.py ``is_private`` network checks. Previously non-private networks (0.0.0.0/0) would return True from this method; now it correctly returns False.
```

Comment:
```suggestion
Fix :attr:`~ipaddress.IPv4Address.is_private` properties in the :mod:`ipaddress` module. Previously non-private networks (0.0.0.0/0) would return True from this method; now they correctly return False.
```
Link to the documented one (https://docs.python.org/3.10/library/ipaddress.html#ipaddress.IPv4Address.is_private)